### PR TITLE
Task2

### PR DESCRIPTION
**Pull request by NataliiaSubotyshyna**

### CODE REVIEW

**Review by SvitlanaLN — CHANGES_REQUESTED** [review]
image
Must be (a+b)/2 , not only (a+b) .

**Review by SvitlanaLN — COMMENTED** [review]
image

It is better  to write 
Console.WriteLine($" grn to usd: {grn_to_usd}\n grn to eur: {grn_to_eur}");

**Comment by NataliiaSubotyshyna** [comment]
@SvitlanaLN ok thank you

**Review by SvitlanaLN — APPROVED** [review]

File: Projects/Team2/Subotyshyna_Nataliia/task_208/task_208/Program.cs [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            double inch = 2.54;
+            double centimeter = 1;
+            
+
+            double res;
+            while (true)
```
Comment (romanukhaav) [inline Projects/Team2/Subotyshyna_Nataliia/task_208/task_208/Program.cs:18]: Is it an infinite cycle? When this cycle should be finished?

**Pull request closed without merging**